sentry/src/test/java/io/sentry/instrumentation/file/SentryFileInputStreamTest.kt

```diff
@@ -0,0 +1,201 @@
+package io.sentry.instrumentation.file
+
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.whenever
+import io.sentry.IHub
+import io.sentry.SentryOptions
+import io.sentry.SentryTracer
+import io.sentry.SpanStatus
+import io.sentry.SpanStatus.INTERNAL_ERROR
+import io.sentry.TransactionContext
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+import java.io.File
+import java.io.FileDescriptor
+import java.io.FileInputStream
+import java.io.IOException
+import kotlin.test.assertEquals
+
+class SentryFileInputStreamTest {
+
+    class Fixture {
+        val hub = mock<IHub>()
+        val sentryTracer = SentryTracer(TransactionContext("name", "op"), hub)
+
+        fun getSut(
+            tmpFile: File? = null,
+            activeTransaction: Boolean = true,
+            delegate: FileInputStream? = null,
+            fileDescriptor: FileDescriptor? = null,
+            sendDefaultPii: Boolean = false
+        ): SentryFileInputStream {
+            tmpFile?.writeText("Text")
+            whenever(hub.options).thenReturn(
+                SentryOptions().apply {
```

Comment:
cheaper having a SentryOptions as an attribute of Fixture, and flipping only the flag